src/biblebot/paths.py

```
@@ -16,38 +27,133 @@
+        return True
+
+    try:
+        target.parent.mkdir(parents=True, exist_ok=True)
+        shutil.move(str(legacy), str(target))
+        logger.info(
+            "Migrated %s from %s to %s",
+            legacy_name,
+            legacy.parent,
+            target.parent,
+        )
+        return True
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)